crates/aptos/src/update/tool.rs

```diff
@@ -14,6 +14,7 @@ pub enum UpdateTool {
     Aptos(AptosUpdateTool),
     Revela(RevelaUpdateTool),
     Movefmt(FormatterUpdateTool),
+    InstallProverDependency(ProverDependencyInstaller),
```

Comment:
Let's call it `ProverDependencies` or just `Prover`. None of the other commands have `install`, the `update` command is already presumed to install if not present, update if so.

Reply:
done